SwiftTweaks/FloatingTweakGroupViewController.swift

```diff
@@ -72,13 +72,13 @@ internal final class FloatingTweakGroupViewController: UIViewController {
 
 	internal static let height: CGFloat = 168
 	internal static let margins: CGFloat = 5
-	private static let minimizedWidth: CGFloat = 30
+	fileprivate static let minimizedWidth: CGFloat = 30
```

Comment:
All these aren't needed outside of this type, so they should be `private` instead.